src/utilities/types.ts

```diff
@@ -113,10 +114,11 @@ export const TYPES: { [key: string]: Partial<IFieldConfig> } = {
     },
   },
   hidden: {
-    editComponent: Antd.Input,
-    editProps: { type: 'hidden' },
+    editComponent: Hidden,
+    fieldConfigProp: true,
```

Comment:
How did this change help resolve the bug? Was it because hiding the edit component still left the space? And what do the newly added properties do?

Reply:
@karen-ka That is a great question. I'm going to try to write the answer in a way I can add it as a comment to Hidden.tsx:

> Most components are automatically wrapped with a lot of boiler-plate form code in FormItem.tsx. Components that want more control over this can pass skipFieldDecorator and then use FormItem themselves. For hidden inputs, we want almost no wrapping at all so the component doesn't render anything on the page. To do this, we use skipFieldDecorator to opt out of boiler-plate then fieldConfigProp to get some advanced props ( like fieldConfig ) so that we can do some of what FormItem does for ourselves below:

Let me know if you have more questions that doesn't answer, or if something about the answer wasn't clear / could be clearer.

Reply:
That makes sense, thank you!

Reply:
Thanks!